app/controllers/hostgroups_controller.rb

```diff
@@ -76,11 +75,10 @@ def destroy
   end
 
   def current_parameters
-    Taxonomy.as_taxonomy @organization, @location do
-      inherited_parameters = Hostgroup.find(params['hostgroup_parent_id']).inherited_params(true) if params['hostgroup_parent_id']
```

Comment:
It looks like this might have had a different behaviour before - actually showing domain, OS, taxonomy parameters etc that are inherited from the parent host group?  Not sure that it was a very consistent behaviour though.